docs/aurora/compiling-and-linking/aurora-programming-models.md

```diff
@@ -32,6 +32,24 @@ The oneAPI compiler flags For some build systems (e.g. `cmake`), it may be neces
 
 OpenCL is supported, but does not require specific compiler flags per-se as the offloaded kernels are JIT-compiled. One does need to link against the OpenCL library `-lOpenCL`. Abstraction programming models, such as Kokkos, can be built on top of these programming models.
 
+# Aurora Programming Environment
+
+## Overview
+
+The Aurora Programming Environment (Aurora PE) is Aurora's default software environment and consists of the OneAPI SDK, MPICH, and the Spack PE. The Aurora PE is loaded in the user environment through a default module set. Alternative versions of the Aurora PE, as well as Aurora PE components not loaded by default, are available through the module interface. Each version of the PE may have multiple compilers, runtimes, and/or MPICH installations; these are interchangeable within a particular PE version. In addition to the OneAPI and MPICH installations, the Aurora PE contains the Spack PE, which contains a myriad of general and scientific computing software. See the [Spack PE page](./applications-and-libraries/libraries/spack-pe.md) for more details.
+
+## Switching PE or SDK versions
+
+Aurora PE and OneAPI SDK versions can be viewed with `module avail` and switched with `module load` commands. The Aurora PE modules utilize Lmod's hierarchical module system to allow seamless switching between versions of the Aurora PE and its components. For example, if the user loads a different version of the `oneapi` SDK, the other modules in the Aurora PE, such as `intel_compute_runtime`, `mpich`, and `spack-pe-gcc`, will be reloaded to guarantee compatibility. In short, the hierarchical modulefile system ensures that the module environment is self-consistent with minimal user input.
+
+## Aurora Programming Environment and /soft
+
+The Aurora PE is installed in `/opt/aurora` and is mounted as a read-only squashfs. `/soft` is also available to provide software not present in the Aurora PE. Modules in `/soft/modulefiles` are not part of the default environment in order to avoid filesystem metadata overhead, which can have significant performance impacts. Users wishing to use software in `/soft` will need to first run `module use /soft/modulefiles` to access the modules.
```

Comment:
```suggestion
The Aurora PE is installed in `/opt/aurora` and is mounted as a read-only SquashFS. `/soft` is also available to provide software not present in the Aurora PE. Modules in `/soft/modulefiles` are not part of the default environment in order to avoid filesystem metadata overhead, which can have significant performance impacts. Users wishing to use software in `/soft` will need to first run `module use /soft/modulefiles` to access the modules.
```